catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/test/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexerTest.java

```diff
@@ -300,30 +316,146 @@ public void testUpdateIndexSourceUnavailableException() throws Exception {
 
   @Test
   public void testUpdateIndexWithGeoEntryList() throws Exception {
-    List<GeoEntry> geoEntries = Collections.singletonList(GEO_ENTRY);
+    List<GeoEntry> geoEntries = singletonList(GEO_ENTRY);
     geoNamesCatalogIndexer.updateIndex(geoEntries, false, progressCallback, GEO_ENTRY_DATA_SOURCE);
     verify(catalogFramework, times(1)).create(any(CreateRequest.class));
-    verify(catalogProvider, times(0)).delete(any(DeleteRequest.class));
-    verify(catalogFramework, times(0)).query(any(QueryRequest.class));
+    verify(catalogProvider, never()).delete(any(DeleteRequest.class));
+    verify(catalogFramework).query(queryCaptor.capture());
+    assertThatLastQueryIsBuildSuggesterIndexQuery();
   }
 
   @Test
   public void testUpdateIndexWithGeoEntryListWithCreate() throws Exception {
     setupTestForCreate();
 
-    List<GeoEntry> geoEntries = Collections.singletonList(GEO_ENTRY);
+    List<GeoEntry> geoEntries = singletonList(GEO_ENTRY);
     geoNamesCatalogIndexer.updateIndex(geoEntries, true, progressCallback, GEO_ENTRY_DATA_SOURCE);
     verify(catalogFramework, times(1)).create(any(CreateRequest.class));
     verify(catalogProvider, times(1)).delete(any(DeleteRequest.class));
-    verify(catalogFramework, times(2)).query(any(QueryRequest.class));
+    verify(catalogFramework, times(3)).query(queryCaptor.capture());
+
+    assertThatLastQueryIsBuildSuggesterIndexQuery();
+  }
+
+  @Test
+  public void testSetImportLocation() throws Exception {
```

Comment:
These two tests aren't related to changes I made in this PR but I added them to cover untested branches so I wouldn't have to lower the BRANCH limit.